### ISSUE TITLE
[css-overflow-3] Reserved scroller space between content edge and padding edge

### ISSUE THREAD

**Issue by gitspeaks** [w3c/csswg-drafts#11288]
The [CSS Overflow 3 spec, Section 4.1](https://www.w3.org/TR/css-overflow-3/#scrollbar-sizing) states:

> When reserving space for a scrollar placed at the edge of an element’s box, the reserved space is inserted between the inner border edge and the outer padding edge. 

Test:

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <style>
      article {
        border: 10px solid red;
        width: 200px;
        height: 200px;
        position: relative;
        overflow: scroll;
        padding-right: 50px;
        
      }
      aside {
        position: absolute;
        top: 0;
        right: 0;
      }
    </style>
  </head>
  <body>
    <article>
      <aside>ASIDE</aside>
      <p>Text1</p>
      <p style="text-align: right;">Text2</p>
    </article>
  </body>
</html>
```

In Chrome DevTools, the scrollbar appears between the **content box edge** (inner edge) and the **padding box edge**, not between the **inner border edge** and **outer padding edge** as specified.

![scroller](https://github.com/user-attachments/assets/e2b5d030-deef-43f7-a89f-e3d00ffe71be)

**Comment by gitspeaks** [comment]
Also, the terms "inner" and "outer" in [Section 4.1](https://www.w3.org/TR/css-overflow-3/#scrollbar-sizing) seem irrelevant in this context. See PR #11287 for details.

**Comment by Loirooriol** [comment]
The devtools don't say where the edges are...

![](https://github.com/user-attachments/assets/88874e68-1b36-4a09-b324-92286235e60d)

**Comment by gitspeaks** [comment]
>CSS Overflow is implicitly patching the definition: https://drafts.csswg.org/css-overflow-3/#scrollbar-sizing

CSS Overflow uses undefined terms, so it’s unclear what it is actually trying to convey.

If I understand you correctly, you’re saying that: "the gutter is placed between the padding area and the border area". If that’s the case, it should be explicitly stated as such, rather than relying on "inner" and "outer" terms, which are meaningless within the context of the current vocabulary.

**Comment by Loirooriol** [comment]
Yes, that's my understanding of CSS Overflow, which I agree should be more explicit.

**Comment by fantasai** [comment]
@gitspeaks Merged your PR, is there anything else that needs to be fixed here?

**Comment by gitspeaks** [comment]
@fantasai  Seems complete, thanks.
